src/modules/core/components/Popover/Popover.tsx

```diff
@@ -116,31 +120,42 @@ const Popover = ({
 
   const lastIsOpenProp = usePrevious(isOpenProp);
 
+  const close = useCallback(
+    (data?: any, modifiers?: { cancelled: boolean }) => {
+      if (openTimeoutRef.current) {
+        clearTimeout(openTimeoutRef.current);
+      }
+      if (closeTimeoutRef.current) {
+        clearTimeout(closeTimeoutRef.current);
+      }
+      _setIsOpen(false);
+      if (typeof onClose == 'function') onClose(data, modifiers);
```

Comment:
Please use brackets. It makes the code much more readable

```suggestion
      if (typeof onClose == 'function') {
        onClose(data, modifiers)
      };
```

Reply:
All fixed! Please check again when possible. Thanks!